### PR TITLE
Update Ibexa Cloud CLI

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | All
| Edition       | All

Move from `ibexa_cloud` alias to `ibexa_cloud` binary.

Move `ibexa_cloud` from alias of P.sh CLI `platform` with env var setup to real Ibexa Cloud CLI. 

- `ibexa_cloud` binary was fully released recently, and is available on https://cli.ibexa.co/
- `ibexa_cloud` alias was introduced in #2064, was lacking an update, and is now deprecated or nonfunctional.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md [hunk 1]
```diff
@@ -412,8 +412,9 @@ Typically, you can add a `gw` to the hostname and use nslookup to find it.
 You can also use the [Platform.sh CLI command](https://docs.platform.sh/administration/cli.html) to find [the endpoint](https://docs.platform.sh/domains/steps/dns.html):
```
Comment (adriendupuis) [inline docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md:412]: TODO: Update

Comment (adriendupuis) [reply docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md]: Done in 13e8762f

**Review by adamwojs — APPROVED** [review]

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 2]
```diff
@@ -85,16 +85,22 @@ When you are done with configuration, push your project to the Platform.sh remot
...
 !!! caution
 
-    If you want to use the [Platform.sh CLI](https://docs.platform.sh/administration/cli.html), you have to set it up for [[= product_name_cloud =]].
-    Consider using the following alias `ibexa_cloud` with a dedicated setup instead of `platform`:
-    ```bash
-    alias ibexa_cloud="PLATFORMSH_CLI_SESSION_ID=ibexa_cloud \
-        PLATFORMSH_CLI_API_URL=https://api.cloud.ibexa.co \
-        platform"
-    ```
+    Do not use Platform.sh CLI (`platform`) but [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
```
Comment (MagdalenaZuba) [inline docs/getting_started/install_on_ibexa_cloud.md:100]:
```suggestion
    Do not use Platform.sh CLI (`platform`), instead, use the [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
```

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 3]
```diff
@@ -85,16 +85,22 @@ When you are done with configuration, push your project to the Platform.sh remot
...
+
+    To install Ibexa Cloud CLI, follow https://cli.ibexa.co/ "Installation instructions".
+
+    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) whre you just have to replace `platform` with `ibexa_cloud`.
```
Comment (MagdalenaZuba) [inline docs/getting_started/install_on_ibexa_cloud.md:104]:
```suggestion
    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) where you have to replace `platform` with `ibexa_cloud`.
```

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 4]
```diff
@@ -85,16 +85,22 @@ When you are done with configuration, push your project to the Platform.sh remot
...
+
+    If you have previously set up an alias to use Platform.sh CLI with Ibexa Cloud, it is outdated. Please, remove the alias and install Ibexa Cloud CLI instead.
```
Comment (MagdalenaZuba) [inline docs/getting_started/install_on_ibexa_cloud.md:106]:
```suggestion
    If you have previously set up an alias to use Platform.sh CLI with Ibexa Cloud, it is outdated. Remove the alias and install Ibexa Cloud CLI instead.
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 5]
```diff
@@ -85,16 +85,23 @@ When you are done with configuration, push your project to the Platform.sh remot
...
-    ```bash
-    alias ibexa_cloud="PLATFORMSH_CLI_SESSION_ID=ibexa_cloud \
-        PLATFORMSH_CLI_API_URL=https://api.cloud.ibexa.co \
-        platform"
-    ```
+    Do not use Platform.sh CLI (`platform`), instead, use the [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
+
+    To install Ibexa Cloud CLI, follow https://cli.ibexa.co/ "Installation instructions".
+
+    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) where you have to replace `platform` with `ibexa_cloud`.
```
Comment (dabrt) [inline docs/getting_started/install_on_ibexa_cloud.md:104]:
```suggestion
    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) but you have to replace `platform` with `ibexa_cloud`.
```

Is it what is meant here? Tha
```suggestion
    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use), but you have to replace `platform` with `ibexa_cloud`.
```

Is it what is meant here? That you have to replace `platform` with `ibexa_cloud` in commands and docs?

File: docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md [hunk 6]
```diff
@@ -409,12 +409,9 @@ Typically, you can add a `gw` to the hostname and use nslookup to find it.
    Address:  1.2.3.4
 ```
 
-You can also use the [Platform.sh CLI command](https://docs.platform.sh/administration/cli.html) to find [the endpoint](https://docs.platform.sh/domains/steps/dns.html):
+You can also use the [Ibexa Cloud CLI](https://cli.ibexa.co/) (which has same command as the Platform.sh CLI) to find [the endpoint](https://docs.platform.sh/domains/steps/dns.html):
```
Comment (dabrt) [inline docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md:412]:
```suggestion
You can also use the [Ibexa Cloud CLI](https://cli.ibexa.co/) (which has the same command as the Platform.sh CLI) to find [the endpoint](https://docs.platform.sh/domains/steps/dns.html):
```

File: docs/infrastructure_and_maintenance/clustering/ibexa_cloud_and_ddev.md [hunk 7]
```diff
@@ -11,19 +11,13 @@ Two ways are available to run an [[= product_name_cloud =]] project locally with
...
 !!! note
 
-    The following examples will use [Platform.sh CLI](https://docs.platform.sh/administration/cli.html) set up for Ibexa Cloud
-    thanks to the following dedicated alias `ibexa_cloud`:
-    ```bash
-    alias ibexa_cloud="PLATFORMSH_CLI_SESSION_ID=ibexa_cloud \
-        PLATFORMSH_CLI_API_URL=https://api.cloud.ibexa.co \
-        platform"
-    ```
+    The following examples will use [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
```
Comment (dabrt) [inline docs/infrastructure_and_maintenance/clustering/ibexa_cloud_and_ddev.md:14]:
```suggestion
    The following examples use [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
```

**Review by dabrt — COMMENTED** [review]
Please see my comments and implement if you agree.

File: docs/getting_started/install_on_ibexa_cloud.md (hunk 5 above)
Comment (adriendupuis) [reply docs/getting_started/install_on_ibexa_cloud.md:104]: Yes, this is still the right meaning.
Use the same doc but replace `platform` with `ibexa_cloud`.

**Pull request merged**